payments_service/solitude/tests/test.py

```diff
@@ -0,0 +1,102 @@
+from nose.tools import eq_, raises
+from slumber.exceptions import HttpClientError, HttpServerError
+
+from payments_service.base.tests import SolitudeTest, WithDynamicEndpoints
+
+from .. import SolitudeBodyguard
+
+
+class TestSolitudeBodyguard(SolitudeTest, WithDynamicEndpoints):
+
+    def setUp(self):
+        super(TestSolitudeBodyguard, self).setUp()
+
+        class DefaultEndpoint(SolitudeBodyguard):
+            methods = ['get', 'post']
+            # Hook up a random (and unrealistic) Solitude endpoint.
+            resource = 'services.status'
+
+        self.endpoint(DefaultEndpoint)
+
+        # If a test forgets to define a mock, apparently Docker eats
+        # 100% CPU :( These should always be overridden.
+        self.solitude.services.status.get.return_value = {}
+        self.solitude.services.status.post.return_value = {}
```

Comment:
For me its when you try to serialize a mock into DRF, I hacked this here: https://github.com/mozilla/solitude/pull/384/files#diff-9ee95e79b11dc9bcf5b00655e989a446R20